browser/resources/settings/email_aliases_page/styles.ts

```diff
@@ -0,0 +1,183 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import styled from 'styled-components'
+import { color, spacing } from '@brave/leo/tokens/css/variables'
+import Button from '@brave/leo/react/button'
+
+export const BraveIconCircle = styled.div`
+  align-items: center;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: 2.5em;
+  border-radius: 50%;
+  border: #E3E3E8 1px solid;
+  display: flex;
+  min-height: 4.5em;
+  justify-content: center;
+  margin-inline-end: 1.5em;
+  min-width: 4.5em;
+  flex-grow: 0;
+`
+
+export const BraveIconWrapper = styled.div`
+  transform scale(2);
+  display: inline-block;
+`
+
+export const Card = styled.div`
+  background-color: ${color.container.background};
+  border: none;
+  overflow: hidden;
+  padding: ${spacing.l} ;
+`
+
+export const Row = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+`
+
+export const Col = styled.div`
+  display: flex;
+  flex-direction: column;
+`
+
+export const AccountRow = styled(Row)`
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 0px 20px;
+`
+
+export const SignupRow = styled(Row)`
+  justify-content: space-between;
+  align-items: start;
+`
+export const MainEmailTextContainer = styled(Col)`
+  justify-content: center;
+  cursor: default;
+  user-select: none;
+`
+
+export const MainEmail = styled.div`
+  font-size: 130%;
+  font-weight: 600;
+  padding-bottom: 6px;
+`
+
+export const MainEmailDescription = styled.div`
+  font-size: 115%;
+`
+
+export const ManageAccountLink = styled.a`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  font-size: 120%;
```

Comment:
is there an exact font-size that's used in the designs? I haven't seen us use percentage sizes much in b-c

Reply:
fixed